spec/models/round_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Round do
+  it 'should have a valid factory' do
+    k = FactoryBot.create(:round, :fall)
+    k.start_date = Time.new(k.session.year, 9)
+    k.end_date = Time.new(k.session.year, 12, 31)
+    k.save!
+    FactoryBot.create(:round, :winter)
```

Comment:
Does it make sense to have rounds without start/end dates? =/ I think those should be required as well.